File: mobile-app/lib/services/ai_service.dart
import 'api_service.dart';
class AIServiceMobile {
  final ApiService _api = ApiService();
  Future<dynamic> predictLevel(Map<String, dynamic> data) async => await _api.post('/ai/predict-level', data);
  Future<dynamic> getRecommendation() async => await _api.get('/ai/recommendation');
  Future<dynamic> getWeakSkills() async => await _api.get('/ai/weak-skills');
}

File: web-frontend/src/services/aiService.js
import api from './api';

export const aiService = {
  predictLevel: async (data) => (await api.post('/ai/predict-level', data)).data,
  getRecommendation: async () => (await api.get('/ai/recommendation')).data,
  getWeakSkills: async () => (await api.get('/ai/weak-skills')).data,
};
